### ISSUE TITLE
Add more bindings inherited from Tex Input Method?

### ISSUE THREAD

**Issue by favonia** [isovector/cornelis#128]
Emacs's Agda mode inherits many bindings from the Tex Input Method, and I found that many of these inherited bindings are missing. While I can certainly just add these bindings in my own configuration file, I wonder if it's part of the design goal to make `agda.vim` "practically complete"? A few notable cases include things like `\alpha` and `\lambda` (though `\Ga` and `\Gl` of course work). What personally affects me are `\^+` and `\u{}`. Note that these bindings are available in the VS Code Agda extension, too.

**Comment by isovector** [comment]
Cornelis' bindings come from a hack and slash hand parsing of https://github.com/agda/agda/blob/master/src/data/emacs-mode/agda-input.el, which doesn't appear to have any of the bindings you're missing.

I'm happy to add bindings from any other file you can find, or accept a PR adding them!

**Comment by favonia** [comment]
@isovector The tricky part is that Agda inherits the Tex Input method (and then tweaks the bindings):

https://github.com/agda/agda/blob/master/src/data/emacs-mode/agda-input.el#L150-L180

Unfortunately I don't use Emacs... (and that's why I'm here). I believe the eventual source is [latin-ltx.el](https://github.com/emacs-mirror/emacs/blob/master/lisp/leim/quail/latin-ltx.el) which (sometimes) parses Unicode names. Simulating the Lisp code might be too much work, though. When tracking down the issue @plt-amy kindly [dumped the keybindings](https://gist.github.com/plt-amy/4dd5ab2a8003662f2d7a6998ad9ebe7d) using
```lisp
(let ((dm (list 'decode-map)))
  (quail-build-decode-map (list (quail-map)) "" dm 0 5120 '()))
```
I also found [keymap.js](https://github.com/banacorn/agda-mode-vscode/blob/master/asset/keymap.js) from the VS Code Agda mode that is a finite state machine, but it is unfortunately [outdated.](https://github.com/banacorn/agda-mode-vscode/issues/171)

---

I guess the "ultimate" hack is to run an Emacs lisp script to dump the _eventual_ key bindings in the Agda mode and reformat the "usable" parts into Cornelis commands. :grin:

**Comment by isovector** [comment]
How's this list look? https://github.com/joom/latex-unicoder.vim/blob/master/autoload/unicoder.vim

**Comment by isovector** [comment]
Should be fixed in 86174c3eea973f8469743d744c643bcf164fd62d. Let me know if that doesn't end up working for you!

**Comment by favonia** [comment]
@isovector Thank you for your fast patch!

I might have caused some misunderstanding about Amy's file though---it's dumping the eventual list of key bindings, which includes all bindings in agda-input.el as well. To see this, `\Ga` is currently defined twice. On the other hand, maybe it doesn't hurt to define the same thing twice.
